components/gsma-authenticators/src/main/java/com/wso2telco/gsma/authenticators/voice/VoiceIVRFutureCallback.java

```diff
@@ -0,0 +1,88 @@
+package com.wso2telco.gsma.authenticators.voice;
+
+import com.wso2telco.gsma.authenticators.AuthenticatorException;
+import com.wso2telco.gsma.authenticators.Constants;
+import com.wso2telco.gsma.authenticators.DBUtils;
+import com.wso2telco.gsma.authenticators.util.BasicFutureCallback;
+import com.wso2telco.ids.datapublisher.model.UserStatus;
+import com.wso2telco.ids.datapublisher.util.DataPublisherUtil;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.apache.http.HttpResponse;
+import org.apache.http.util.EntityUtils;
+import org.codehaus.jettison.json.JSONException;
+import org.codehaus.jettison.json.JSONObject;
+import org.wso2.carbon.identity.application.authentication.framework.context.AuthenticationContext;
+
+import java.io.IOException;
+import java.sql.SQLException;
+
+/**
+ * Created by sheshan on 5/24/17.
+ */
+public class VoiceIVRFutureCallback extends BasicFutureCallback {
+
+    private static Log log = LogFactory.getLog(VoiceIVRFutureCallback.class);
+    private static final String IS_FLOW_COMPLETED = "isFlowCompleted";
+    private final String UNKNOWN_USER = "UNKNOWN_USER";
+    private final String ACTIVE = "ACTIVE";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this final field static too. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.wso2telco.com:9000/coding_rules#rule_key=squid%3AS1170)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "ACTIVE" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.wso2telco.com:9000/coding_rules#rule_key=squid%3AS00116)